siliconcompiler/tools/openroad/scripts/sc_place.tcl

```diff
@@ -2,7 +2,7 @@
 # Global Placement (without considering IO placements)
 #######################
 
-if { $openroad_gpl_enable_skip_io } {
+if { $openroad_gpl_enable_skip_io && [sc_design_has_placeable_ios] } {
```

Comment:
Shouldn't this be the negation?

Reply:
This call to global_placement only makes sense if the IOs are still placeable, otherwise the next call to pin placement doesn't do much, but I can reorganize this to make it easier to see.

Reply:
Oh now I see. I didn't realize this step would **only** get executed when the IOs are present.